--- /usr/share/emacs/site-lisp/emacs-mozc/mozc.el.orig	2024-03-18 05:06:35.259060646 +0900
+++ /usr/share/emacs/site-lisp/emacs-mozc/mozc.el	2024-03-18 05:10:38.649250691 +0900
@@ -123 +123 @@
-  (require 'cl))
+  (require 'cl-lib))
@@ -394 +394 @@
-    (let ((key (case basic-type
+    (let ((key (cl-case basic-type
@@ -398,31 +398,31 @@
-                 ('eisu-toggle 'eisu)
-                 ('hiragana-katakana 'kana)
-                 ('next 'pagedown)
-                 ('prior 'pageup)
-                 ('kp-decimal 'decimal)
-                 ('kp-0 'numpad0)
-                 ('kp-1 'numpad1)
-                 ('kp-2 'numpad2)
-                 ('kp-3 'numpad3)
-                 ('kp-4 'numpad4)
-                 ('kp-5 'numpad5)
-                 ('kp-6 'numpad6)
-                 ('kp-7 'numpad7)
-                 ('kp-8 'numpad8)
-                 ('kp-9 'numpad9)
-                 ('kp-delete 'delete)  ; .
-                 ('kp-insert 'insert)  ; 0
-                 ('kp-end 'end)        ; 1
-                 ('kp-down 'down)      ; 2
-                 ('kp-next 'pagedown)  ; 3
-                 ('kp-left 'left)      ; 4
-                 ('kp-begin 'clear)    ; 5
-                 ('kp-right 'right)    ; 6
-                 ('kp-home 'home)      ; 7
-                 ('kp-up 'up)          ; 8
-                 ('kp-prior 'pageup)   ; 9
-                 ('kp-add 'add)
-                 ('kp-subtract 'subtract)
-                 ('kp-multiply 'multiply)
-                 ('kp-divide 'divide)
-                 ('kp-enter 'enter)
+                 (eisu-toggle 'eisu)
+                 (hiragana-katakana 'kana)
+                 (next 'pagedown)
+                 (prior 'pageup)
+                 (kp-decimal 'decimal)
+                 (kp-0 'numpad0)
+                 (kp-1 'numpad1)
+                 (kp-2 'numpad2)
+                 (kp-3 'numpad3)
+                 (kp-4 'numpad4)
+                 (kp-5 'numpad5)
+                 (kp-6 'numpad6)
+                 (kp-7 'numpad7)
+                 (kp-8 'numpad8)
+                 (kp-9 'numpad9)
+                 (kp-delete 'delete)  ; .
+                 (kp-insert 'insert)  ; 0
+                 (kp-end 'end)        ; 1
+                 (kp-down 'down)      ; 2
+                 (kp-next 'pagedown)  ; 3
+                 (kp-left 'left)      ; 4
+                 (kp-begin 'clear)    ; 5
+                 (kp-right 'right)    ; 6
+                 (kp-home 'home)      ; 7
+                 (kp-up 'up)          ; 8
+                 (kp-prior 'pageup)   ; 9
+                 (kp-add 'add)
+                 (kp-subtract 'subtract)
+                 (kp-multiply 'multiply)
+                 (kp-divide 'divide)
+                 (kp-enter 'enter)
@@ -826 +826 @@
-                   (case (mozc-protobuf-get segment 'annotation)
+                   (cl-case (mozc-protobuf-get segment 'annotation)
@@ -1257 +1257 @@
-              (incf scrolled-lines))
+              (cl-incf scrolled-lines))
@@ -1643 +1643 @@
-    (case (process-status proc)
+    (cl-case (process-status proc)
